cmake/ConfigFileSetting.cmake

```diff
@@ -164,8 +164,17 @@ configure_file("${PROJECT_SOURCE_DIR}/src/sundials/sundials_config.h.in"
                "${PROJECT_BINARY_DIR}/src/sundials/sundials_config.h" @ONLY)
 configure_file("${PROJECT_SOURCE_DIR}/share/lib/nrn.defaults.in"
                "${PROJECT_BINARY_DIR}/share/nrn/lib/nrn.defaults" @ONLY)
-file(COPY ${PROJECT_SOURCE_DIR}/share/lib/nrnunits.lib
-     DESTINATION ${PROJECT_BINARY_DIR}/share/nrn/lib)
+set(nrnunits_start "")
+set(nrnunits_stop "")
+
+configure_file(${PROJECT_SOURCE_DIR}/share/lib/nrnunits.lib.in
+               ${PROJECT_BINARY_DIR}/share/nrn/lib/nrnunits.lib @ONLY)
+
+set(nrnunits_start "R\"qwerty(")
```

Comment:
Could use a comment. Took me a while (and looking at the constructed embedded_nrnunit.lib file) to realize the reason for the start/stop idiom.